projects/go-lib/src/lib/components/go-table/go-table.component.html

```diff
@@ -33,27 +37,46 @@ <h5 class="go-heading-5 go-heading--no-margin" *ngIf="tableTitle">{{ tableTitle
         </tr>
       </thead>
       <tbody class="go-table__body">
-        <tr *ngFor="let item of setDisplayData()"
-            class="go-table__body-row">
-          <td class="go-table__body-column"
-              width="60"
-              *ngIf="localTableConfig.selectable">
-            <input type="checkbox"
-                   [checked]="isRowSelected(item)"
-                   (change)="selectionChange($event, item)"
-                   [name]="'selection_' + item[localTableConfig.selectBy]"/>
-          </td>
-          <td *ngFor="let col of columns"
-              class="go-table__body-column"
-              [attr.width]="col.width">
-            <ng-container *ngIf="!col.goTableCell; else columnCellOutlet">
-              {{ col.getFieldData(item) }}
-            </ng-container>
-            <ng-template #columnCellOutlet>
-              <ng-container *ngTemplateOutlet="col.goTableCell; context: { $implicit: item }"></ng-container>
-            </ng-template>
-          </td>
-        </tr>
+        <ng-container *ngFor="let item of setDisplayData()">
+          <tr class="go-table__body-row">
+            <td class="go-table__body-column"
+                [ngClass]="{ 'go-table__body-row--expanded': item.showDetails }"
+                width="40"
+                *ngIf="details">
+              <go-icon-button
```

Comment:
So looking at this in the UI, the padding on the `icon-button` is making the row heights much taller than they probably should be. Do you think you could try to find a way around this?

Reply:
I think I fixed it by adding a class that offsets the padding of the row by the amount that's on the `icon-button`